### PR TITLE
[VIRTS-1964] Ability Storage

### PR DESCRIPTION
**Pull request by nopfor**
## Description

Implements https://github.com/mitre/caldera/pull/2120

- Add executors to created abilities
- Create link with executor
- Check link executor name from executor, not ability

## Type of change

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How Has This Been Tested?

See: https://github.com/mitre/caldera/pull/2120

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: app/gameboard_api.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 from aiohttp_jinja2 import template
 
 from app.objects.c_ability import Ability
+from app.objects.secondclass.c_executor import Executor
```
Comment (bworrell) [inline app/gameboard_api.py]: nit: imports should be lexicographically ordered within their respective group

Comment (nopfor) [reply app/gameboard_api.py]: Check the updated order.

**Review by bworrell — COMMENTED** [review]
I haven't tested yet but generally this LGTM. Will test more when core caldera stabilizes.

File: app/gameboard_svc.py [hunk 2]
```diff
@@ -121,7 +121,7 @@ async def _add_detection_to_operation(self, op, link_pid, verify_type):
         self.file_svc.write_result_file(str(link.id), self.encode_string(str(link_pid)))
 
     def _create_and_setup_verification_link(self, ability, link_pid):
-        link = Link(command=None, paw='gameboard_detection', ability=ability)
+        link = Link(command=None, paw='gameboard_detection', ability=ability, executor=ability.executors[0])
```
Comment (bworrell) [inline app/gameboard_svc.py]: I haven't tested this yet but does this need to be updated? e.g. `next(ability.executors)`

**Review by mrengstrom — APPROVED** [review]

**Pull request merged**